test_cli.py

```diff
@@ -1415,35 +1374,37 @@ def stress(
     wait(_run_command([cl, 'run', 'ping -c 1 google.com', '--request-network']), 0)
 
 
-# TODO: can't do this test until we can pass in another CodaLab instance.
-@TestModule.register('copy', default=False)
+@TestModule.register('copy')
 def test(ctx):
     """Test copying between instances."""
     source_worksheet = current_worksheet()
 
-    with temp_instance() as remote:
-        remote_worksheet = remote.home
-        _run_command([cl, 'work', remote_worksheet])
+    with remote_instance(ctx.second_instance) as remote:
 
-        def check_agree(command):
+        def compare_output(command):
```

Comment:
Maybe `compare_output_across_instances`?

Reply:
Yes, that's more explicit. I will update.